src/api/Nhs.Appointments.Api/Functions/RemoveUserFunction.cs

```diff
@@ -21,6 +21,8 @@ public class RemoveUserFunction(
     IMetricsRecorder metricsRecorder)
     : BaseApiFunction<RemoveUserRequest, RemoveUserResponse>(validator, userContextProvider, logger, metricsRecorder)
 {
+    private readonly IUserContextProvider _userContextProvider = userContextProvider;
```

Comment:
Are these required?

Reply:
This is the warning they generate: 
<img width="794" alt="image" src="https://github.com/user-attachments/assets/8a3a6dcb-655a-40d3-ab9a-0d2ae1e4e2a8">
I'm unsure how else to resolve this warning as I'll be honest I don't 100% understand what it means. I'll do some googling and get my head round it, but for expediency I've reverted these changes. Shall I leave 4 warnings unresolved and tackle in a future PR?